.github/workflows/build.yaml

```diff
@@ -52,11 +52,17 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Get current day
```

Comment:
It's not obvious why this is needed, can we have comments explaining what's going on?